ensemble/ctf/manager.py

```diff
@@ -8,7 +8,7 @@
 from .utils import load_ctf, save_ctf
 
 
-CTF_EXTENSION = '.ctf'
+CTF_EXTENSION = u'.ctf'
```

Comment:
How do you feel about `from __future__ import unicode_literals`? I think the test coverage of this module is probably good enough to consider it.

Reply:
The package itself is small enough that i'd prefer just using the `unicode_literals` import in all modules.

Reply:
Agreed